bigquery/start.py

```diff
@@ -0,0 +1,224 @@
+import os.path
+import requests, json
+
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = ["https://www.googleapis.com/auth/bigquery"]
+
+WEATHER_API_KEY = "YOUR_KEY"
+
+
+def create_dataset(creds, project_id, dataset_id):
+    try:
+        service = build("bigquery", "v2", credentials=creds)
+        service.datasets().insert(
+            projectId=project_id,
+            body={
+                "datasetReference": {
+                    "projectId": project_id,
+                    "datasetId": dataset_id,
+                }
+            },
+        ).execute()
+    except HttpError as err:
+        print(err)
+
+
+def create_table(creds, project_id, dataset_id, table_id, table_schema):
+    try:
+        service = build("bigquery", "v2", credentials=creds)
+        service.tables().insert(
+            projectId=project_id,
+            datasetId=dataset_id,
+            body={
+                "tableReference": {
+                    "projectId": project_id,
+                    "datasetId": dataset_id,
+                    "tableId": table_id,
+                },
+                "schema": {"fields": table_schema},
+            },
+        ).execute()
+    except HttpError as err:
+        print(err)
+
+
+def create_city_geo(creds, project_id, dataset_id, table_id, city):
```

Comment:
This explains the need for the API key that I commented on above. However it's not yet clear to me what this function does. On this note, function doc strings are a must!
Along the same lines, these days you want to include type annotations in Python code.